telegram-bot-api/src/Telegram/Bot/API/InlineMode.hs

```diff
@@ -34,13 +34,28 @@ data InlineQuery = InlineQuery
   , inlineQueryLocation :: Maybe Location -- ^ For bots that require user location, sender location
   , inlineQueryQuery    :: Text -- ^ Text of the query, up to 256 characters
   , inlineQueryOffset   :: Text -- ^ Offset of the results to be returned, can be controlled by bot
-  , inlineQueryChatType :: Maybe ChatType -- ^ Type of the chat, from which the inline query was sent. Can be either “sender” for a private chat with the inline query sender, “private”, “group”, “supergroup”, or “channel”. The chat type should be always known for requests sent from official clients and most third-party clients, unless the request was sent from a secret chat.
```

Comment:
Could you please extend `ChatType` instead of introducing `InlineChatType`?

Reply:
I've thought about it, my reasons were: `ChatType` which is in `Chat` can not be `sender`: 

> Type of chat, can be either “private”, “group”, “supergroup” or “channel”

So if someone is pattern matching against this field now he would get non-exhaustive patterns, and will also have to handle this non-existent case. This way i was trying to minimize damage to existing code which uses the library.  

But, yes, I can also see the reasons against it.

Reply:
Okay, as far as I see no usages of `ChatType`. Could you please remove it in favour of `InlineChatType` then?